<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="main_menu_buttons_form">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>151</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>User : Ali Paksoy</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="main_customer_button">
        <property name="text">
         <string>Customer
Operations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_currency_button">
        <property name="text">
         <string>Currency
Operations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_money_transfer_button">
        <property name="text">
         <string>Money
Transfers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_bill_button">
        <property name="text">
         <string>Bills
Paying</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_insurance_button">
        <property name="text">
         <string>Insurance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="main_investment_button">
        <property name="text">
         <string>Investment
Transactions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Main_logout">
        <property name="text">
         <string>LogOut</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="main_customer_ops_form">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Customer Operations</string>
    </property>
    <widget class="QGroupBox" name="customer_ops_buttons_form">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="cs_add_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add
Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cs_remove_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove
Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cs_credit_check_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit
Check</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cs_lend_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Lend</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cs_credit_payment_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit
Payment</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cs_remove_customer_form">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="title">
      <string>remove_customer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Name Surname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>National Id:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="cs_removeform_remove_button">
        <property name="text">
         <string>Cancel
Contract</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="cs_removeform_name_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="cs_removeform_id_input"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="cs_credit_check_form">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string>Credit_check</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="1">
       <widget class="QPushButton" name="cs_creditcheckform_check_button">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>National Id:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Full Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cs_creditcheckform_name_input"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cs_creditcheckform_it_input"/>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="cs_credit_payment_form">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string>Credit_payment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="cs_creditpaymentform_amount_input"/>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Full Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>National Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cs_creditpaymentform_name_input"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cs_creditpaymentform_id_input"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="cs_creditpaymentform_pay_button">
        <property name="text">
         <string>Pay</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="cs_Lend__form">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Lend</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Full Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="cs_lendform_name_input"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="cs_lendform_id_input"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>National Id:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="cs_lendform_amont_input"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="cs_lendform_lend_button">
        <property name="text">
         <string>Lend</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="cs_customer_register_form">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="title">
      <string>add_customer</string>
     </property>
     <widget class="QPushButton" name="cs_registerform_add_button">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>print contract</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cs_registerform_confirm_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm
Customer
Contract</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>135</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="cs_registerform_fullname_input"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cs_registerform_id_input"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cs_registerform_address_input"/>
       </item>
       <item>
        <widget class="QComboBox" name="cs_registerform_nation_input">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>TC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EU</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cs_registerform_tel_input"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>87</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name Surname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>National Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nationality:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Tel No:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="main_currency_ops_form">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Currency Operations</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Sell</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Euro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>US Dollar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ruble</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="main_money_ops_form">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string>Money Transfers</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>To:</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>151</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_18"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>161</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Iban:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>National Id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Account Number:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>From:</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>151</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_16"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_17"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>161</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Full Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>National Id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Amount:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="main_bills_form">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>bills</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Electric</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Water</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Gas</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="main_insurance_form">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string>Insurance</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="main_investment_transactions_form">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>InvestmentTransactions</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string>Main_Page</string>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
